build/ci/github-actions/cache.md

```diff
@@ -151,6 +151,81 @@ jobs:
 ```
 {% endraw %}
 
+### Cache mounts
+
+BuildKit doesn't preserve cache mounts in the GitHub Actions cache by default.
+If you wish to put your cache mounts into GitHub Actions cache and reuse it
+between builds, you can use a workaround provided by two third-party actions:
+
+- `overmindtech/buildkit-cache-dance/extract`
+- `overmindtech/buildkit-cache-dance/inject`
+
+These GitHub Actions creates temporary containers to extract and inject the
+cache mount data with your Docker build steps.
+
+The following example shows how to use this workaround with a Go project.
+
+```yaml
+name: ci
+on: push
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: YOUR_IMAGE
+          tags: |
+            type=ref,event=branch
+            type=ref,event=pr
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
+
+      - name: Go Build Cache for Docker
+        uses: actions/cache@v3
+        with:
+          path: go-build-cache
+          key: ${{ runner.os }}-go-build-cache-${{ hashFiles('**/go.sum') }}
+
+      - name: inject go-build-cache into docker
+        uses: overmindtech/buildkit-cache-dance/inject@main
```

Comment:
use commit sha for third-party actions:

```suggestion
        uses: overmindtech/buildkit-cache-dance/inject@621d3f1980d1e9e6f31fc471687bf342c6f8611a
```

Reply:
Hm, even in docs? How do we make sure we keep this up to date?

Reply:
I thought about it and decided to keep `main` since this is only an example. If users actually decide to use this, I think it's up to them to pin a sha or release-version if they wish.

Reply:
If we keep main and the action introduced something malicious users will complain on us first imo. It's just an extra safety nothing more. We can add a comment explaining why we use a commit sha. Maybe using an include across our pages using third-party GHAs.